rasa/utils/tensorflow/model_data_utils.py

```diff
@@ -49,6 +52,8 @@ def featurize_training_examples(
         A list of attribute to features.
         A dictionary of attribute to feature sizes.
     """
+    if (type is not None) and (type not in [SEQUENCE, SENTENCE]):
+        raise ValueError(f"Unknown type {type}")

```

Comment:
Use a better exception type.

Reply:
What would you suggest here? I can't find any exception.py containing something that looks more suitable than ValueError here

Reply:
Not sure.. I've asked about this on the engineering channel - https://rasa-hq.slack.com/archives/C01HHMR4X8S/p1627632564005400